doc/developingCubeBrowsers/interactions_2.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Linking up Cube Pickers and Plot Customization Options\n",
+    "\n",
+    "This is an example of a set of workflow options linked up with ipywidgets.  The options in this demo are for preparing one plot of an iris cube.\n",
+    "\n",
+    "There are some assumptions in this workflow:\n",
+    "1. Data path is set to iris.sample_data_path\n",
+    "2. Coordinates for plot axes will be longitude vs latitude\n",
+    "3. Plot type will be line contour (plot customizations are applicable to this plot type)\n",
+    "\n",
+    "This notebook demostrates the links between some elements of a selection process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import IPython.display\n",
+    "import ipywidgets\n",
+    "import iris\n",
+    "\n",
+    "import cube_browser"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "ipynb = IPython.get_ipython()\n",
+    "if ipynb is not None:\n",
+    "    ipynb.magic(u\"%matplotlib notebook\")\n",
+    "    ipynb.magic(u\"%autosave 0\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This filepath is a directory containing the set of options for the first selection box.\n",
+    "path = '/home/h06/cbosley/Projects/iris-sample-data/sample_data'\n",
```

Comment:
I had trouble getting this to work, it kept telling me it required one argument.